src/Tickets/Emails/Assets.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Handles registering and setup for assets on Tickets Emails.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+use \tad_DI52_ServiceProvider;
+
+/**
+ * Class Assets.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Assets extends tad_DI52_ServiceProvider {
```

Comment:
I don't think we need this as a service provider. It can be a plain class.

Reply:
Assets and Hooks should be service providers.